frame/staking/src/benchmarking.rs

```diff
@@ -634,6 +644,81 @@ benchmarks! {
 	verify {
 		assert!(!Validators::<T>::contains_key(controller));
 	}
+
+	rebag {
+		// The most expensive case for this call:
+		//
+		// - It doesn't matter where in the origin bag the stash lies; the number of reads and
+		//   writes is constant. We can use the case that the stash is the only one in the origin
+		//   bag, for simplicity.
+		// - The destination bag is not empty, because then we need to update the `next` pointer
+		//   of the previous node in addition to the work we do otherwise.
+
+		use crate::voter_bags::{Bag, Node};
+
+		let make_validator = |n: u32, balance_factor: u32| -> Result<(T::AccountId, T::AccountId), &'static str> {
+			let (stash, controller) = create_stash_controller::<T>(n, balance_factor, Default::default())?;
+			whitelist_account!(controller);
+
+			let prefs = ValidatorPrefs::default();
+			// bond the full value of the stash
+			let free_balance = T::Currency::free_balance(&stash);
+			Staking::<T>::bond_extra(RawOrigin::Signed(stash.clone()).into(), free_balance)?;
+			Staking::<T>::validate(RawOrigin::Signed(controller.clone()).into(), prefs)?;
+
+			Ok((stash, controller))
+		};
+
+		// Clean up any existing state.
+		clear_validators_and_nominators::<T>();
+
+
+		// stash controls the node account
+		let (stash, controller) = make_validator(USER_SEED, 100)?;
+
+		// create another validator with 3x the stake
+		let (other_stash, _) = make_validator(USER_SEED + 1, 300)?;
+
+		// update the stash account's value/weight
+		let other_free_balance = T::Currency::free_balance(&other_stash);
+		T::Currency::make_free_balance_be(&stash, other_free_balance);
+		Staking::<T>::bond_extra(RawOrigin::Signed(stash.clone()).into(), other_free_balance)?;
```

Comment:
```suggestion
```

you already call `make_validators(_, 300)`, so `other_stash` should have 3 times stake bonded.

Reply:
No, we need this. We're not nooping on `other_stash`; we're _changing_ `stash`'s bond to match `other_stash`, which is much greater, so that we know that `stash` will be misplaced.